modules/bismark_methXtract.cfmod

```diff
@@ -64,13 +64,6 @@ The bismark_methXtract module runs the bismark_methylation_extractor script.
 This reads in a bisulfite read alignment results file produced by the Bismark bisulfite
 mapper and extracts the methylation information for individual cytosines in CpG, CHG 
 and CHH context.\n".
-# Specifying the pbat or single_cell parameters in a pipeline
```

Comment:
I agree that irrelevant comments can go… We have changed the way we treat PBAT or single-cell data to remove them with Trim Galore right from the start (4N, 6N, 9N, 12N, youNameN), and the align and extract everything instead of using the --ignore options. The only thing that singlecell does for us right now is to run --non_directional mode.

From: Phil Ewels [mailto:notifications@github.com]
Sent: 19 January 2015 13:35
To: ewels/clusterflow
Subject: Re: [clusterflow] Merging Simon's BI changes back to 0.4 development branch (#27)


In modules/bismark_methXtract.cfmod<https://github.com/ewels/clusterflow/pull/27#discussion-diff-23159919>:

> @@ -64,13 +64,6 @@ The bismark_methXtract module runs the bismark_methylation_extractor script.

>  This reads in a bisulfite read alignment results file produced by the Bismark bisulfite

>  mapper and extracts the methylation information for individual cytosines in CpG, CHG

>  and CHH context.\n".

> -# Specifying the pbat or single_cell parameters in a pipeline

I guess we should probably also delete the commented out code below that this refers to. I forget why this isn't included in this module any more..?

—
Reply to this email directly or view it on GitHub<https://github.com/ewels/clusterflow/pull/27/files#r23159919>.
The Babraham Institute, Babraham Research Campus, Cambridge CB22 3AT Registered Charity No. 1053902.
The information transmitted in this email is directed only to the addressee. If you received this in error, please contact the sender and delete this email from your system. The contents of this e-mail are the views of the sender and do not necessarily represent the views of the Babraham Institute. Full conditions at: www.babraham.ac.uk<http://www.babraham.ac.uk/terms>